webp.mjs

```diff
@@ -297,7 +297,7 @@ class Image {
           break;
       }
       cursor += header.size+1;
-      if (header.size&1) { cursor++; }
+      // if (header.size&1) { cursor++; }
```

Comment:
I'll explaing this change so you can decide what to do with it

I was getting this error:
```bash
(node:3454) UnhandledPromiseRejectionWarning: Error: Unknown image type
    at Function.muxAnim (file:///home/marcelo/www/test/node_modules/node-webpmux/webp.mjs:541:24)
    at Image.muxAnim (file:///home/marcelo/www/test/node_modules/node-webpmux/webp.mjs:178:126)
    at /home/marcelo/www/test/index.js:8:13
```

From my investigation, this switch was never matching any case, therefore the frames didn't had the type filled: 
https://github.com/ApeironTsuka/node-webpmux/blob/9d01fb7f5fd514253c404a4a7c07f800e13b8ad8/webp.mjs#L275

I log the values of `header` and got this values:

```bash
{ fourCC: 'ALPH', size: 57 }
{ fourCC: 'P8 >', size: 1342177318 }
{ fourCC: 'ALPH', size: 57 }
{ fourCC: 'P8  ', size: 268435494 }
{ fourCC: 'ALPH', size: 57 }
{ fourCC: 'P8 ,', size: 4026531878 }
{ fourCC: 'ALPH', size: 57 }
{ fourCC: 'P8 �', size: 1879048231 }
{ fourCC: 'ALPH', size: 57 }
{ fourCC: 'P8 T', size: 2952790057 }
{ fourCC: 'ALPH', size: 57 }
{ fourCC: 'P8 r', size: 1879048233 }
{ fourCC: 'ALPH', size: 57 }
{ fourCC: 'P8 �', size: 1342177321 }
{ fourCC: 'ALPH', size: 57 }
{ fourCC: 'P8 �', size: 2415919145 }
{ fourCC: 'ALPH', size: 57 }
{ fourCC: 'P8 T', size: 3489660969 }
{ fourCC: 'ALPH', size: 57 }
{ fourCC: 'P8 �', size: 3489660969 }
{ fourCC: 'ALPH', size: 57 }
{ fourCC: 'P8 |', size: 805306409 }
{ fourCC: 'ALPH', size: 57 }
{ fourCC: 'P8 �', size: 3489660969 }
{ fourCC: 'ALPH', size: 57 }
{ fourCC: 'P8 �', size: 4026531882 }
```

So the `fourCC` was never matching the swtich case values, therefore the frame ended up like this:
```js
{
  raw: <Buffer 00 00 00 00 00 00 ff 01 00 ff 01 00 64 00 00 00 41 4c 50 48 39 00 00 00 01 17 20 10 48 c2 dd 5f 60 8d 88 08 07 45 6d a4 40 98 04 67 88 be fe 8d e8 ff ... 10972 more bytes>,
  x: 0,
  y: 0,
  width: 512,
  height: 512,
  duration: 100,
  blend: true,
  dispose: false
}
```